### DISCUSSION TITLE
Does inclusion of a meal event automatically change the physiology to fed?

### DISCUSSION THREAD

**Discussion by Angelefleury** [Open-Systems-Pharmacology/Forum#801] [Questions & Problems] [answered]
Dear all,

I want to do a simple comparison of a fasted versus fed profile in PK Sim. For this I included a meal event at the time of drug administration as sole change (therefore did not consider a change in solubility). Parameters such as gastric pH and transit time (GET) did not change as a consequence of the inclusion of this high caloric meal event.
I then changed the solubility: pH and GET remained unchanged.
From #638 I understood that those parameters would be changed automatically to the reference values under fed conditions. Did I get this wrong and should we change actively those parameters, or did I miss a step?

I thank you very much for any thoughts regarding this.

Best regards
Angèle

**Comment by StephanSchaller** [comment]
Hi Angèle, the parameters do not change visibly in the parameters within the simulation, but will then dynamically change during the simulation (MoBi will provide insights here).
Did the simulation curves change?

Best,
Stephan

**Comment by Angelefleury** [comment]
Hi Stephan,

Thank you very much for your time and efforts to help.

Yes: the shape of the curve changes, with similar AUCinf but a shift to a later Tmax and lower Cmax, suggesting indeed that some changes in GET and/or pH occurs although not visible in PK Sim (again for this exercise I consider the same solubility input value fed or fasted).
In MoBi: unfortunately the changes are not visible here neither (GET = 15min and Stomach pH = 2.00 in both cases). Does the user have a way of monitoring this?
Best wishes
Angèle

**Comment by msevestre** [comment]
@Angelefleury If this is a parameter that you want to monitor, you can simply set the flag "Plot parameter" to true, rerun the simulation and you should be able to see the change of parameters over time

**Comment by Angelefleury** [comment]
Dear @msevestre ,
Thank you very much for the tip. After some attempts it seems that GET are really equal between fed and fasted (the only difference in the simulation is the presence or absence of a meal event):
![image](https://user-images.githubusercontent.com/38788212/115894985-3188e880-a45a-11eb-9abd-6d778ec89a80.png)

so there is definitely a difference between the two curves, but I can't see its origins form the model parameters.
I believe I still miss a step.

**Comment by StephanSchaller** [comment]
Hi Angèle,

the is definitely a difference. @PavelBal can certainly help ;-)

Enjoy the weekend,
Stephan

**Comment by Yuri05** [comment, marked as answer]
@Angelefleury A meal event changes not the GET itself (the GET parameter can be in fact interpreted as "GET **fasted**") but the **normalized velocity of drug transfer from stomach into duodenum**.

If you look at the drug transport from stomach to duodenum, it's defined like this:
![grafik](https://user-images.githubusercontent.com/25061876/115955477-ed97f100-a4f6-11eb-8aa0-261c096ed9c7.png)

`Intestinal transit rate (absolute)` is the **normalized (to the amount of a molecule in stomach) velocity of drug transfer**.

Which is defined **as per default** (= in fasted state) as `1/GET`:

![grafik](https://user-images.githubusercontent.com/25061876/115955576-a2caa900-a4f7-11eb-98b0-5fd13da67fb3.png)

A meal event changes (among other things) this velocity and sets it to a **function**.

![grafik](https://user-images.githubusercontent.com/25061876/115955677-4d42cc00-a4f8-11eb-8210-d1b46cf07bec.png)
This new (Weibull-like) velocity function is based on the parameters of a meal (volume, caloric intake, etc.) and is constructed the way that it's smaller than 1/GET at the beginning and is increasing till it reaches the old 1/GET(fasted) value.

Thus the meal effect "stops" when the normalized velocity is greater than 1/GET. 
Then the normalized velocity is set back to its original formula `1/GET`:
![grafik](https://user-images.githubusercontent.com/25061876/115955808-381a6d00-a4f9-11eb-95c9-fee05f58f18f.png)

**Comment by Angelefleury** [comment]
Dear @Yuri05 , dear all,
I thank you very much for this extensive and very helpful explanation. Navigating through the model it makes sense. It was also tackled as nicely already in #638, sorry for that.
![image](https://user-images.githubusercontent.com/38788212/116540443-709bbb80-a8ea-11eb-967f-d060faeefc3f.png)


Best wishes
